spec/web_helpers.rb

```diff
@@ -1,7 +1,8 @@
-def create_space
+def create_space(name, description="")
```

Comment:
Maybe the default description should be "description" instead of an empty string?